qfdmd/migrations/0029_alter_produit_filieres_rep_and_more.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 5.1.4 on 2025-01-14 13:26
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("qfdmd", "0028_remove_lien_poids"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="produit",
+            name="filieres_rep",
+            field=models.CharField(blank=True, help_text="Filière(s) REP concernée(s)"),
+        ),
+        migrations.AlterField(
+            model_name="produit",
+            name="nom_eco_organisme",
+            field=models.CharField(blank=True, help_text="Nom de l’éco-organisme"),
+        ),
+        migrations.AlterField(
+            model_name="produit",
+            name="synonymes_existants",
+            field=models.TextField(
+                blank=True,
+                help_text="Ce champ est obsolète, il n'est actuellement pas mis à jour automatiquement.",
```

Comment:
Je pense que ça suffira à régler le pb de CI

```suggestion
                help_text=(
                    "Ce champ détermine la position d'un élément dans la liste affichée.<br>"
                    "Les éléments avec un poids plus élevé apparaissent plus bas dans "
                    "la liste.<br>"
                    "Les éléments avec un poids plus faible apparaissent plus haut."
                )
```